go.mod

```diff
@@ -13,6 +13,7 @@ require (
 	github.com/ashanbrown/makezero v0.0.0-20210520155254-b6261585ddde
 	github.com/bkielbasa/cyclop v1.2.0
 	github.com/bombsimon/wsl/v3 v3.3.0
+	github.com/butuzov/ireturn v0.0.0-20210903145654-645c1cbdbffd
```

Comment:
Can you create a tag? Like that bot can handle the updates.
And it's a good practice.

Reply:
Sure, just I wanted to add one more thing before going 0.1.0.